save-plugins/fix-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugins/fix/FixPluginTest.kt

```diff
@@ -159,6 +160,62 @@ class FixPluginTest {
         }
     }
 
+    @Test
+    fun `should resolve placeholders`() {
```

Comment:
And these tests can be moved from plugins into common module, because you are not testing plugin-specific logic here